dnf-testing.py

```diff
@@ -9,10 +9,12 @@
 import sys
 import time
 
-DESTRUCTIVE_TAGS = set([
-    "@destructive",
-    "@no_installroot",
-])
+PROGPATH = os.path.abspath(os.path.dirname(sys.argv[0]))
+
+# add behave tests root to python path so the `consts` module can be imported
+sys.path.append(os.path.join(PROGPATH, 'dnf-behave-tests'))
+import consts
```

Comment:
I'm not sure about adding this much python module hackery... I kind of considered the one in `dnf-behave-tests/common` a necessary evil. Maybe this won't cause issues but maybe we'd be better off just duplicating the two tags here?

Reply:
I tried to avoid duplicity here (although in first versions the DESTRUCTIVE_TAGS was duplicated here). Having tests correctly recognized as destructive is crucial and I just wanted to narrow the space for an error by forgetting to edit it in both places.